modules/ossm-install-ossm-operator.adoc

```diff
@@ -4,36 +4,31 @@
 // - service_mesh/v2x/installing-ossm.adoc
```

Comment:
I think OpenShift Elasticsearch operator is an optional operator. By default, we don't need to install Elasticsearch operator before installing OSSM operators. So it's better you update that step "1. OpenShift Elasticsearch" with "1. (Optional) OpenShift Elasticsearch"